TestProject/gameoflife.php

```diff
@@ -0,0 +1,105 @@
+<?php
+use cellularAutomat\Board;
+use options\Getopt;
+
+
+/**
+ * Function for Autoloader.
+ * @param $className
+ */
+function myAutoload($className) {
+    $classPath = str_replace("\\", "/", $className);
+    require_once(sprintf("%s/%s.php", __DIR__, $classPath));
+}
+
+/**
+ * Excecution of the autoloader
+ */
+spl_autoload_register("myAutoload");
+
+
+/**
+ * Command line parameters
+ */
+$options = new Getopt([['r', "startRandom", Getopt::NO_ARGUMENT, "Starts the board with random cells which are dead/alive"],
+                        ['g', "startGlider", Getopt::NO_ARGUMENT, "Starts the board with a glider which runs through the board"],
+                        ['w', "width", Getopt::REQUIRED_ARGUMENT, "Sets the width of the board"],
+                        ['h', "heigth", Getopt::REQUIRED_ARGUMENT, "Sets the height of the board"],
+                        ['s', "maxSteps", Getopt::NO_ARGUMENT, "Sets the number of the generations"],
```

Comment:
This won't work like this! If you use Getopt::NO_ARGUMENT, the value will be the number of times the parameter was passed. If you try it out you should see that the max-steps is always only 1 no matter which value you pass as parameter.